File: backend/supabaseClient.py
import os
from supabase import create_client, Client
from dotenv import load_dotenv

load_dotenv()

supabaseUrl = os.environ.get("SUPABASE_URL")
supabaseKey = os.environ.get("SUPABASE_KEY")

supabase: Client = create_client(supabaseUrl, supabaseKey)

File: frontend/src/supabaseClient.js
import { createClient } from '@supabase/supabase-js'

const supabaseUrl = 'https://vjuzseukocsdoyqchfiw.supabase.co'
const supabaseKey = 'sb_publishable_tAMnOcSA7Oc3otkud739OQ__P9CyfJH'

export const supabase = createClient(supabaseUrl, supabaseKey)
